zuul.d/jobs.yaml

```diff
@@ -72,6 +72,13 @@
       global_env:
         OS_BRANCH: stable/stein
 
+- job:
+    name: terraform-provider-openstack-v1.19.0-acceptance-test-stein
+    parent: terraform-provider-openstack-acceptance-test-stein
+    description: |
+      Run terraform provider openstack of v1.19.0 acceptance test on stein branch
```

Comment:
not sure when split `terraform-provider-openstack` to `terraform provider openstack` in all the descriptions, I do not think it is reasonable. because the `terraform-provider-openstack` is the repo and project name.
and also, maybe the `on stein branch` is not correct, should be `on the stein branch of OpenStack`.

Reply:
done